src/Uno.UI/UI/Xaml/Controls/SplitView/SplitView.iOS.cs

```diff
@@ -1,78 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Microsoft.UI.Xaml.Media;
-using Microsoft.UI.Xaml.Media.Animation;
-
-namespace Microsoft.UI.Xaml.Controls
-{
-	partial class SplitView
-	{
-		/*
-
-			About the PatchInvalidFinalState:
```

Comment:
I guess we need to make sure this is no longer an issue on iOS.

Maybe @Xiaoy312 will know.

Reply:
not something i had worked on, but @dr1rrb is the author here
it would be difficult to verify now since there is no associated repro